hex/helpers/build

```diff
@@ -10,7 +10,14 @@ fi
 install_dir="$DEPENDABOT_NATIVE_HELPERS_PATH/hex"
 mkdir -p "$install_dir"
 
-mix local.hex --force
+# Initial Hex install - will always be the latest available version
+mix local.hex --force --if-missing
+# Annoyingly, a specific Hex version cannot be specified during the initial install.
+# The only way to pin is to re-install.
+if [ -n "$HEX_VERSION" ]; then
+  mix hex.install "$HEX_VERSION"
+fi
+
 mix archive.install hex nerves_bootstrap --force
```

Comment:
I'm a little uncertain on this... From what I read, the `mix archive.install` is a global install, so doing `mix archive.install hex` seems like it might be once again installing/overwriting the hex install? 🤷 I'll open an issue against the Hex repo and ask for their input...

Reply:
https://github.com/hexpm/hex/issues/962